spec/models/user_spec.rb

```diff
@@ -1194,5 +1194,24 @@
         end
       end
     end
+
+    describe '#military_information', :skip_va_profile do
+      it 'returns an instance of the EMISRedis class' do
+        Flipper.disable('military_information_vaprofile')
```

Comment:
Please don't disable or enable the flipper directly, please do:

`before { allow(Flipper).to receive(:enabled?).with('military_information_vaprofile').and_return(true) }`

otherwise you may hit issues with flaky specs

Reply:
Will do 👍. I'll make a note to update [the docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample) with your recommendation.